ansible/deploy-oonibackend-docker.yml

```diff
@@ -24,6 +77,7 @@
     enable_tcp_web_connectivity: yes
     enable_tcp_bouncer: yes
     enable_tcp_collector: yes
+    oonibackend_bundle: yes
```

Comment:
I guess if I have to nitpick it's a bit confusing that setting this variable to "no" actually doesn't have the expected effect of disabling it, since you just check for the variable being defined.